telegram-bot/main.go

```diff
@@ -3,15 +3,16 @@ package main
 import (
 	"context"
 	"fmt"
+	"github.com/anaskhan96/soup"
+	"github.com/dghubble/go-twitter/twitter"
+	"github.com/dghubble/oauth1"
+	tbot "github.com/go-telegram-bot-api/telegram-bot-api"
 	"log"
 	"math/rand"
 	"os"
 	"strconv"
 	"time"
 
-	"github.com/anaskhan96/soup"
```

Comment:
This shouldn't be removed as it is still a required module for fetching xkcd.

Reply:
Oh, sorry for the spacing thing, also it isn't removed, I just put it above, see Line 6. Want me to put it in the original format?

Reply:
Yes. Please do 😃

Reply:
Done.